data-source/json/test_schema_theme_census.json

```diff
@@ -0,0 +1,68 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "A questionnaire to check schema census theme.",
+    "theme": "census",
+    "description": "A questionnaire to check schema census theme.",
+    "messages": {
+        "NUMBER_TOO_LARGE": "Number is too large",
+        "NUMBER_TOO_SMALL": "Number cannot be less than zero",
+        "INVALID_NUMBER": "Please enter an integer"
+    },
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "sections": [
+        {
+            "id": "default-section",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "lunch-block",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "favourite-lunch",
+                                        "label": "What is your favourite lunchtime food",
+                                        "mandatory": false,
+                                        "q_code": "0",
+                                        "type": "TextField"
+                                    }
+                                ],
+                                "description": "",
+                                "id": "favourite-lunch-question",
+                                "title": "",
+                                "type": "General"
+                            },
+                            "routing_rules": []
+                        },
```

Comment:
While not necessary, it would make more sense for this test schema to ask something like `What is your name?` with a single textfield answer for `Name`.

Reply:
Fixed, thanks!